src/drt/src/TritonRoute.cpp

```diff
@@ -976,6 +953,65 @@ void TritonRoute::checkDRC(const char* filename, int x1, int y1, int x2, int y2)
   reportDRC(filename, markers, requiredDrcBox);
 }
 
+void TritonRoute::stackVias(odb::dbBTerm* bterm,
+                            int top_layer_idx,
+                            int bterm_bottom_layer_idx)
+{
+  odb::dbTech* tech = db_->getTech();
+  odb::dbSet<odb::dbTechVia> vias = tech->getVias();
+  std::map<int, odb::dbTechVia*> default_vias;
+  for (int i = top_layer_idx; i <= bterm_bottom_layer_idx; i++) {
+    for (odb::dbTechVia* via : vias) {
+      if (via->getBottomLayer()->getRoutingLevel() == i) {
+        default_vias[i] = via;
+        break;
+      }
```

Comment:
Grabbing any via on the right layer may not work well.  At a minimum you should take a default via but even then we may need to add more checks (eg https://github.com/The-OpenROAD-Project/OpenROAD/pull/3056)

Reply:
It's not clear how to get the default vias. To get it from DRT, it is necessary to run `initDesign` first, but my new command runs before it is initialized.

We also have this process on grt, but I think this method doesn't fit here:

```
for (odb::dbTechVia* via : vias) {
    odb::dbStringProperty* prop
        = odb::dbStringProperty::find(via, "OR_DEFAULT");

    if (prop == nullptr) {
      continue;
    } else {
      debugPrint(logger_,
                 utl::GRT,
                 "l2v_pitch",
                 1,
                 "Default via: {}.",
                 via->getConstName());
      default_vias[via->getBottomLayer()->getRoutingLevel()] = via;
    }
  }
```

Reply:
Actually, I think it can be simplified. Perhaps we can drop this new command and just run the stack vias process inside drt, without user input. It would be simpler for users and for the implementation, as it can be executed after defining the default vias.